freqtrade/freqtradebot.py

```diff
@@ -95,6 +95,16 @@ def __init__(self, config: Dict[str, Any]) -> None:
         # Protect sell-logic from forcesell and viceversa
         self._sell_lock = Lock()
 
+    def notify_status(self, msg: str) -> None:
```

Comment:
I'm not 100% convinced about moving this to freqtradebot - why not move this as "helper" to `RPCManager`.

On the other hand, it's aligend to where notifications for buy / sells are generated  - which we cannot really move to RPCManager as they depend on methods (e.g. `get_sell_rate()` available from freqtrade.